po/nl.po

```diff
@@ -6,14 +6,14 @@ msgstr ""
 "Project-Id-Version: newsboat 2.7\n"
 "Report-Msgid-Bugs-To: https://github.com/newsboat/newsboat/issues\n"
 "POT-Creation-Date: 2020-02-02 16:53+0300\n"
-"PO-Revision-Date: 2013-10-26 22:20+0100\n"
+"PO-Revision-Date: 2020-02-04 22:01+0100\n"
 "Last-Translator: Erwin Poeze <donnut@outlook.com>\n"
 "Language-Team: Dutch <vertalen@vrijschrift.org>\n"
```

Comment:
You should put your name into `Last-Translator` and `Language-Team` fields, and move Erwin Poeze into `Language-Team` as well. (We don't really have translation teams in Newsboat, so the convention is to list every translator in the team field.)